plugins/MauticCrmBundle/Api/SalesforceApi.php

```diff
@@ -267,7 +310,7 @@ public function getLeads($query, $object)
 
             $getLeadsQuery = 'SELECT '.$fields.' from '.$object.' where LastModifiedDate>='.$query['start'].' and LastModifiedDate<='.$query['end'].$ignoreConvertedLeads;
```

Comment:
Need to force the format of the start and end dates.  It has to be ISO-8601. 

Example, the following will find no results even if there are.

<pre>
# Returns nothing
./app/console -vvv mautic:integration:synccontacts -i Salesforce -d '2016-07-01' -t '2016-08-01'
</pre>

versus this actually working

<pre>
# Return 500+ results
./app/console -vvv mautic:integration:synccontacts -i Salesforce -d '2016-07-01T00:00:00Z' -t '2016-08-01T00:00:00Z'
</pre>